pkgs/top-level/all-packages.nix

```diff
@@ -11419,121 +11419,123 @@ in
      for a specific kernel.  This function can then be called for
      whatever kernel you're using. */
 
-  linuxPackagesFor = kernel: self: let callPackage = newScope self; in rec {
+  linuxPackagesFor = kernel: lib.makeExtensible (self: {
```

Comment:
Come to think of it, we could just do `(self: with self; {` and cut down a lot of the diff.